tools/query_verification/src/main/java/com/google/bigquery/BigQueryManager.java

```diff
@@ -279,8 +281,55 @@ public String getName() {
             StandardSQLTypeName type = fields.get(i).getType().getStandardType();
 
             Object result;
-            // TODO Convert value to appropriate Java type
-            result = value.getStringValue();
+            try {
+                switch (type) {
+                    case BOOL:
+                        // returns boolean
```

Comment:
I think we can remove all these `// returns` comments, the type information is already present